modules/core/shared/src/test/scala/gem/arb/ArbMagnitudeValue.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package arb
+
+import gem.math.MagnitudeValue
+import org.scalacheck._
+import org.scalacheck.Arbitrary._
+import org.scalacheck.Cogen._
+
+trait ArbMagnitudeValue {
```

Comment:
I'm using the class name as the name of the file `ArbMagnitudeValue`. I noted that `Arbitraries`' file names use the orignal name which is a bit confusing when you search by file name

Reply:
Either way is fine with me, but we should be consistent.

Reply:
I'd like to change them just because of the ergonomics of editing. now when I search for a file Angle I get 5 hits

Reply:
Let's do a PR that does nothing but rename files, since it will create gigantic deltas.